src/topology/continuous_function/basic.lean

```diff
@@ -82,7 +82,7 @@ end
 def id : C(α, α) := ⟨id⟩
 
 @[simp] lemma id_coe : (id : α → α) = id := rfl
-lemma id_apply (a : α) : id a = a := rfl
+@[simp] lemma id_apply (a : α) : id a = a := rfl
```

Comment:
I think the linter will reject this, since `id_coe` should fire first?

Reply:
The linter already accepted it, and see the example in the PR text, `id_coe` doesn't fire.

Reply:
See below and [this zulip thread](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/A.20linter.20to.20prevent.20.60x.20.3D.20x.60.20lemmas/near/238022646), `id_coe` is a stupid lemma that states nothing.